src/coreclr/gc/gc.cpp

```diff
@@ -641,6 +641,13 @@ process_sync_log_stats()
 #ifndef DACCESS_COMPILE
 uint32_t g_num_active_processors = 0;
 
+// Note that when a join is no longer used we still keep the values here because
+// tooling already recognized them as having the meaning they were assigned originally.
+// It doesn't break tooling if we stop using them but does if we assign a new meaning 
+// to them. 
+//
+// No longer in use:
+// gc_join_disable_software_write_watch
```

Comment:
maybe add a comment that `gc_join_disable_software_write_watch` is no longer use on line 691 as well?